docs/modules/ROOT/partials/ref_devfile-reference.adoc

```diff
@@ -758,9 +759,10 @@ projects:
   - name: my-go-project
     clonePath: go/src/github.com/acme/my-go-project
     git:
-      location: https://github.com/acme/my-go-project.git
+      remotes: https://github.com/acme/my-go-project.git
```

Comment:
Should be (make similar changes to other remotes references to add `origin` like this example):
```
      remotes: 
        origin: "https://github.com/acme/my-go-project.git"
```

Reply:
Addressed, so resolved convo.